blog/2017-08-10-RealTimePlanningUsingKieServerPart1.adoc

```diff
@@ -0,0 +1,149 @@
+= Optimize your problems using KIE Server, part 1
+:page-interpolate: true
+:awestruct-author: mcimbora
+:awestruct-layout: blogPostBase
+:awestruct-tags: [feature, howto, kie server]
+
+KIE Server provides an execution environment separated from your client application.
+Run it on the same local network as your client, or in the cloud, depending on your needs.
+The server capabilities are exposed via JMS/REST interface, with the latter making the content easily consumable
+by non-Java clients.
+This is extremely useful if you want to leverage OptaPlanner outside Java ecosystem.
+In the 7 release stream, we worked to improve OptaPlanner/KIE Server integration.
+Apart from API improvements I am happy to announce the server now supports real-time planning.
```

Comment:
Real-time planning, that sounds interesting. That deserves a concrete example.
On the other hand, that's the topic of "part 2". I don't think "part 2" should be named "part 2": it should stand on it's own as the article about real-time planning with KIE server.